scalafix/rules/src/main/scala/fix/v0_14_0/FixQuery.scala

```diff
@@ -13,14 +16,26 @@ class FixQuery extends SemanticRule("FixQuery") {
   import FixQuery._
 
   private def isQuery(term: Tree)(implicit doc: SemanticDocument): Boolean =
-    term.symbol.info.exists(_.overriddenSymbols.exists(HasQueryMatcher.matches))
+    term.symbol.info.exists(_.overriddenSymbols.exists(HasQueryMethodMatcher.matches))
+
+  private def isQueryFormat(term: Tree)(implicit doc: SemanticDocument): Boolean =
+    term.symbol.info.map(_.signature).exists {
+      case ClassSignature(_, parents, _, _) =>
+        parents.exists {
+          case TypeRef(_, symbol, _) if HasQueryMatcher.matches(symbol) => true
+          case _                                                        => false
+        }
+      case _ => false
+    }
 
   override def fix(implicit doc: SemanticDocument): Patch = {
     doc.tree.collect {
-      case t @ q"$qual.$fn" if HasQueryMatcher.matches(fn) =>
+      case t @ q"$qual.$fn" if HasQueryMethodMatcher.matches(fn) =>
         Patch.replaceTree(t, q"$qual.queryRaw".syntax)
       case t @ q"$qual.query" if isQuery(t) =>
         Patch.replaceTree(t, q"$qual.queryRaw".syntax)
+      case t @ q"$qual.query.format" if isQueryFormat(qual) =>
+        Patch.replaceTree(t, q"$qual.queryRaw.format".syntax)
```

Comment:
Shouldn't we try to directly use `query(..)` in that case ?
```suggestion
        Patch.replaceTree(t, q"$qual.query".syntax)
```

Reply:
I simplified it to single `case` matching `query` directly 👍